dev_utils/src/utils.rs

```diff
@@ -2,21 +2,36 @@ use std::ops::{Add, Sub};
 
 use rand::{thread_rng, Rng};
 use rand_distr::Uniform;
+use num_traits::float::FloatCore;
 
 // random array that samples between min and max of T
 pub fn get_random_array<T>(n: usize, min_value: T, max_value: T) -> Vec<T>
-where
-    T: Copy + rand::distributions::uniform::SampleUniform,
+    where
+        T: Copy + rand::distributions::uniform::SampleUniform,
```

Comment:
These formatting changes were applied automatically when run through the formatter. If this is undesired, we should configure it, but from what I understand Rust is pretty opinionated here.

Reply:
I do not have extremely strong opinions about formatting - leaving this as is, is certainly fine for me!  
Of course, if this bugs you - you can always elaborate on why so and how we should then format this :)

Reply:
Couldn't care less about formatting. Just did whatever was default. Glad we're on the same page.